kotlin-html/build.gradle.kts

```diff
@@ -87,6 +87,7 @@ tasks.withType(KotlinCompile::class)
             )
         }
     }
+*/
```

Comment:
Why comment this out?

Reply:
hi @ScottPierce I' not sure what the Osx platform reports during compilation but on my erlatively up-to-date intellij linux configuration this generates a big warning about experimental features.  the codebase doesn't appear to be doing anything forward-looking at this point in time by commenting this and frankly, i forgot about this until after commit since nothing broke.

Reply:
Turning on the new inference fixed a compilation error I was seeing initially. I un-commented this code in one of my PRs today.